src/utils.py

```diff
@@ -42,7 +42,10 @@ def get_mapping_set(
 ) -> pd.DataFrame:
     """Load up MedGen mapping set (MedGenIDMappings.txt), with some modifications."""
     # Read
-    df = pd.read_csv(inpath, sep='|').rename(columns={'#CUI': 'xref_id'})
+    df = pd.read_csv(inpath, sep='|').rename(columns={
+        '#CUI_or_CN_id': 'xref_id',
+        '#CUI': 'xref_id'  # 2024/05/19: MedGen Renamed to "#CUI_or_CN_id". Leaving this in case they change back.
```

Comment:
## The bug fix
MedGen renamed a field, so I needed to update the code to match.

Reply:
I agree with the change on [Line 46](https://github.com/monarch-initiative/medgen/blob/build-errors/src/utils.py#L46) as the MedGen format or data model as you mention has changed. 

I would not have left Line 47 in and would prefer to review the new MedGen data format/model if it changes again in the future. However, you do have the reason for this nicely commented and so far the remaining code would be ok (albeit possibly redundant) if a `CUI` column were added back _and_ only included CUIs.

Reply:
Actually, you bring up a good point. What if they do add a `CUI` column and it's not the same as it was before; like it only includes CUIs and not CNs, as it was doing before? Well, then me leaving this line in here would create problems. So I just removed it now!

> would prefer to review the new MedGen data format/model if it changes again in the future.

If you would like I could upload a copy of the mapping file for you to take a look at, but I didn't keep  the old version from before the column name changed.